src/tools/clippy/tests/ui/arithmetic_side_effects.stderr

```diff
@@ -1,706 +1,719 @@
 error: arithmetic operation that can potentially result in unexpected side-effects
-  --> $DIR/arithmetic_side_effects.rs:304:5
+  --> tests/ui/arithmetic_side_effects.rs:304:5
```

Comment:
Those changes are really weird. Let me look into this.

Reply:
On my local machine, I got `$DIR` when blessing. I pushed that change. Let's see if Rust CI agrees :thinking: